common/org.testeditor.dsl.common/.classpath

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
+	<classpathentry exported="true" kind="lib" path="lib/commons-lang3-3.4.jar"/>
```

Comment:
Since commons-lang3 is available in Eclipse Orbit this is not a valid approach here. It should be added to the target platform.